src/entries/popup/components/Autocomplete/index.tsx

```diff
@@ -25,8 +44,19 @@ export interface AutocompleteProps {
   borderColor?: BoxStyles['borderColor'];
   placeholder?: string;
   open: boolean;
+  autoFocus?: boolean;
+  tabIndex: number;
 }
 
+export type customNetworkInfo = {
+  rpcUrl: string;
+  chainId: number;
+  decimals: number;
+  symbol: string;
+  explorerUrl: string;
+  testnet: boolean;
+};
+
```

Comment:
I was trying to make this component agnostic to custom networks in case we need to reuse in the future. 
Anways, I don't mind keeping it like this and refactor later if needed.